sofl/preferences.py

```
@@ -270,6 +271,26 @@ def update_cover_callback(manager: SgdbManager) -> None:
             }
         )
 
+        # Synchronize theme switch with force-theme setting
+        theme = shared.schema.get_string("force-theme")
+        self.force_theme_switch.set_active(theme == "dark")
+
+        def on_theme_switch(row, _param):
+            shared.schema.set_string(
```

Pylint (3.3.7)

[error] Instance of 'MockSettings' has no 'set_string' member; maybe 'get_string'?

(E1101)